libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -2747,6 +2747,15 @@ string YulUtilFunctions::conversionFunction(Type const& _from, Type const& _to)
 		});
 	}
 
+	if (_from.category() == Type::Category::Array)
```

Comment:
Can you turn this and the above into `else if` to make the structure clearer?

Reply:
Did like that